.github/workflows/container_build_template.yml

```diff
@@ -0,0 +1,193 @@
+      # if container_tags is not defined, then image_tags will be auto-generated
+      # if container_tags is latest, it will be added to auto-generated image_tags
+      # if container_tags is defined and not latest, it will be used as is
+      - name: Build image tags 🏷️
+        id: build-tags
+        run: |
+          if [ -z "${{ inputs.container_tags }}" ]; then
+            case ${{ inputs.container_name }} in
+              base)
+                echo "image_tags=python${{ inputs.python_version }}" >> $GITHUB_OUTPUT
+                ;;
+              dev)
+                echo "image_tags=python${{ inputs.python_version }}" >> $GITHUB_OUTPUT
+                ;;
+              *)
+                echo "image_tags=python${{ inputs.python_version }}-avd-${{ github.ref_name }}" >> $GITHUB_OUTPUT
+                ;;
+            esac
+          else
+            if [ "${{ inputs.container_tags }}" == "latest" ]; then
+              case ${{ inputs.container_name }} in
+                base)
+                  echo "image_tags=latest,python${{ inputs.python_version }}" >> $GITHUB_OUTPUT
+                  ;;
+                dev)
+                  echo "image_tags=latest,python${{ inputs.python_version }}" >> $GITHUB_OUTPUT
+                  ;;
+                *)
+                  echo "image_tags=latest,python${{ inputs.python_version }}-avd-${{ github.ref_name }}" >> $GITHUB_OUTPUT
+                  ;;
+              esac
+            else
+              echo "image_tags=${{ inputs.container_tags }}" >> $GITHUB_OUTPUT
+            fi
+          fi
+
+      - name: Generate from_image and from_variant parameters 🐍
+        id: generate-from-parameters
+        # Logic:
+        # 1) only python-slim images are supported at the moment
+        # 2) base image must be built first from official python image
+        # 3) all other images will be built from the base image
+        # 4) from_image and from_variant can be passed transparently to the build step
+        #    but that is not implemented currently
+        run: |
+          if [ "${{ inputs.from_image }}" == "python" ]; then
+            if [ "${{ inputs.container_name }}" == "base" ]; then
+              echo "from_image=${{ inputs.from_image }}" >> $GITHUB_OUTPUT
+              echo "from_variant=${{ inputs.python_version }}-slim-bullseye" >> $GITHUB_OUTPUT
+            else
+              echo "from_image=ghcr.io/${{ github.repository }}/base" >> $GITHUB_OUTPUT
+              echo "from_variant=python${{ inputs.python_version }}" >> $GITHUB_OUTPUT
+            fi
+          else
+            echo "Only python-slim images are supported at the moment. Exiting."
+            exit 1
+          fi
+
+      - name: find ansible-core version 🔥
+        id: find-ansible-core-version
+        run: |
+          ANSIBLE_CORE_VERSION=$(cat ansible_collections/arista/avd/requirements-dev.txt| grep ansible-core)
```

Comment:
probably best to use this file: https://docs.ansible.com/ansible/latest/dev_guide/developing_collections_structure.html#meta-directory-and-runtime-yml

Reply:
Not 100% sure what file are your referring to? The point is, I have to build the ansible core version from a file in a specific branch. That, I think is the easiest way. But if there is a better file, happy to consider. It must be part of the branch though and not something external.

Reply:
I'll just remove that.